src/state/actions/actionTypes.js

```diff
@@ -0,0 +1 @@
+export const CREATE_NEW_ARTICLE = "CREATE_NEW_ARTICLE";
```

Comment:
As we discussed as a group, let's remove the redux files from this pr since they are not being used. If we end up needing to refactor for functionality we later want to implement, we can add these back in then. :)